p2p/src/rpc.rs

```diff
@@ -66,6 +72,7 @@ trait P2pRpc {
 
     /// Add the address to the reserved nodes list.
     /// The node will try to keep connections open to all reserved peers.
+    /// A reserved peer is a peer that you trust and you want your node to remain connected to, no matter what they do.
```

Comment:
Would be nice to put a blank line between the brief high-level description and additional details for rustdoc. (Multiple occurrences of this)